### PR TITLE
Some Label changes

### PR DESCRIPTION
**Pull request by cornfeedhobo**
This PR fixes an issue I found where `SizeHint` was suggesting too small of a size when word wrapping was triggered.

Included is a `GetText` function to retrieve the current text contents.

### CODE REVIEW

File: label.go [hunk 1]
```diff
@@ -75,6 +78,11 @@ func (l *Label) heightForWidth(w int) int {
 	return len(strings.Split(wordwrap.WrapString(l.text, uint(w)), "\n"))
 }
 
+// GetText returns the text content of the label.
+func (l *Label) GetText() string {
```
Comment (marcusolsson) [inline label.go]: I think this should be just `Text()` to be consistent with the naming scheme, e.g., `TextEdit.Text()`.

Comment (cornfeedhobo) [reply label.go]: Sounds good to me. I'll update that.

**Comment by cornfeedhobo** [comment]
Change made. Thanks.

**Comment by marcusolsson** [comment]
Looks great! Thank you!

**Pull request merged**